candle-wasm-examples/yolo/src/model.rs

```diff
@@ -623,16 +623,31 @@ fn iou(b1: &Bbox, b2: &Bbox) -> f32 {
     i_area / (b1_area + b2_area - i_area)
 }
 
-pub fn report(pred: &Tensor, img: DynamicImage, w: usize, h: usize) -> Result<Vec<Vec<Bbox>>> {
+pub fn report(
+    pred: &Tensor,
+    img: DynamicImage,
+    w: usize,
+    h: usize,
+    conf_threshold: f32,
+    iou_threshold: f32,
+) -> Result<Vec<Vec<Bbox>>> {
     let (pred_size, npreds) = pred.dims2()?;
     let nclasses = pred_size - 4;
+    let conf_threshold = Some(conf_threshold)
```

Comment:
I don't get this error on my env, I'm on OSX, is there a better way to do this ````used `unwrap_or()` on `Some` value```` ?

Reply:
`unwrap_or` only applies when the option is `None` so doing `Some(a).unwrap_or(b)` is actually equivalent to `a` :) Not sure how your setup is but hopefully clippy should be easy to install on macos too (or at least it was for me :) )

Reply:
I was trying to set a default value as well as constrain to 0-1, is there a better way to do this? thanks

Reply:
As the parameter of the function is `u32`, it will always be present so there is no notion of default value. If you want to make it optional, you can change it to `Option<u32>` in the argument list?
The min/max looks reasonable, slightly more idiomatic is `.clamp(0., 1.)`.

Reply:
ok that's great feedback, I'll assume there's always a preset value then, and clamp it.